lib/connextclient/ConnextClient.ts

```diff
@@ -692,6 +694,44 @@ class ConnextClient extends SwapClient {
     });
   }
 
+  public withdraw = async ({
+    all,
+    currency,
+    amount: argAmount,
+    destination,
+    fee,
+  }: WithdrawArguments): Promise<string> => {
+    if (fee) {
+      // TODO: allow overwriting gas price
```

Comment:
Let's create issues for the TODOs in this PR.

Reply:
Opened https://github.com/ExchangeUnion/xud/issues/1856